modules/metering-uninstall.adoc

```diff
@@ -3,25 +3,34 @@
 // * metering/metering-uninstall.adoc
 
 [id="metering-uninstall_{context}"]
-= Uninstalling metering from {product-title}
+= Uninstalling metering from your namespace
 
-You can remove metering from your cluster.
+Uninstall metering from your namespace by removing the MeteringConfig resource and deleting the `openshift-metering` namespace.
```

Comment:
Are you really just removing it from your namespace, or from the entire cluster? Slightly confused because you say from your namespace, but then say to specifically delete the openshift-metering namespace.

Reply:
> Are you really just removing it from your namespace, or from the entire cluster? Slightly confused because you say from your namespace, but then say to specifically delete the openshift-metering namespace.

@bergerhoffer Good question. @timflannagan1 Would it make more sense to title this section as "Uninstalling a metering namespace" and for the introduction to be:

"Uninstall your metering namespace, for example the `openshift-metering` namespace, by removing the MeteringConfig resource and deleting the metering namespace."

Reply:
The reference to the `MeteringConfig` resource is a single, namespace-scoped custom resource metering is configured to use. Deleting this resource first, and then deleting the metering namespace, allows the metering operator to clean up resources that it provisioned safely. You can get the same outcome by deleting the namespace, but you potentially risk a namespace stuck in a terminating state.

Reply:
So in the case here, I think we're interested in documenting a path for uninstalling a single metering installation in a single namespace. In the case that's the only metering installation in the cluster, you can safely remove the cluster-scoped metering CRDs.